lib/sink.js

```diff
@@ -239,7 +239,9 @@ module.exports = class SinkGCS extends EventEmitter {
     async has(fileName) {
         assert(fileName, '"fileName" is missing');
         await this[ready];
-        const result = await this[getGcsFile](fileName).exists();
+        const result = await this[getGcsFile](fileName).exists({
+            validation: false,
```

Comment:
What was the motivation for this? I've been hunting through the gcs docs but getting fed up trying to track down where they heck options is explained.

Reply:
The asset server crashed, meaning finn.no wqs without js and css

Reply:
How was this able to bring down the frontpage? Only optimistic bundling was changed today and the frontpage doesnt use it. What am I missing?

Reply:
oh right, the server crashed. derp

Reply:
Do you have a reference to the docs for this option (or any options) Those docs are terrible? I feel like a worthless incapable human being after spending 10 mins trying to find out what the options are.

Reply:
I found it by looking through the source code the last time. I'll see if I can dig it up